provers/evm-prover/.env.example

```diff
@@ -0,0 +1,5 @@
+export CELESTIA_NODE_AUTH_TOKEN=
```

Comment:
any reason why you added a separate .env.example?

Reply:
It should use the common `.env`